### PR TITLE
First attempt at a pretty view

### PR DESCRIPTION
**Pull request by KrzyszD**
Added a grid to holder containers, added an input with username and password, added filler text.

Feedback would be nice.

or knowing what I would need on this page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,41 @@
 import React, { Component } from 'react';
 
+// styles uses grid to have 2 columns, I think personally it looks better and easier to read
+//But again, what do I know what we need
```
Comment (jc3m) [inline src/App.js]: Leave comments like these on the actual PR

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    font-size: 1.25em;
+    line-height: 1.5em;
+    box-sizing: border-box;
+}
+
+html{
+    /* Change this value if you want to have a different min width */
+    min-width: 1000px;
+    background-color: #fbfbfb;
+}
```
Comment (jc3m) [inline src/styles.css]: Don't forget the newline at the end of a file!

File: src/App.js [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <h1>Admin Portal</h1>
+        <h1 className="adminHeader">Admin Portal</h1>
+
+        <div className="divList grid">
+          <div className="divListElement">
```
Comment (jc3m) [inline src/App.js]: Let's rename this class to something like `grid-element`

Also, the naming convention for css classes is to use dashes like above

**Comment by KrzyszD** [comment]
I fixed the syntax, but unable to merge the stuff in myself

**Comment by jc3m** [comment]
Ok, I'll go ahead and merge it for you

**Pull request merged**